src/App.jsx

```diff
@@ -13,15 +13,15 @@ export default function App() {
       <ProvideAuth>
         <Router>
           <Switch>
-            <Route exact path="/">
+            <Route path="/">
               <Home />
             </Route>
             <Route path="/login">
               <Login />
             </Route>
-            <Route path="/treehouse/members">
```

Comment:
Nice work adding your PrivateRoute here by using the react context to make state accessible to your entire application